### DISCUSSION TITLE
Broken link

### DISCUSSION THREAD

**Discussion by JohnBoy6666** [community/community#28425] [Other Feature Feedback, Questions, & Ideas]
Not sure how to contact someone in Github.

This link https://github.com/prof7bit/TorChat has a broken link within it:

"https://github.com/prof7bit/TorChat/downloads"

**Comment by radar-blip** [comment]
Thats someones respository, not githubs.

Contact the repository owner and let him/her know through Issues.

Edit:  Nevermind. I did it for you.

**Comment by JohnBoy6666** [comment]
How do I do that?

On Aug 4, 2022, at 8:09 PM, David Lawson ***@***.***> wrote:


Thats someones respository, not githubs.

Contact the repository owner and let him/her know through Issues.

**Reply by radar-blip** [reply]
Go to the repository in question, and click on the Issues tab, then click on New Issue... fill in the blanks. 😃
